clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/input/kafka/ttl/TTLResolutionPolicy.java

```diff
@@ -0,0 +1,60 @@
+package com.linkedin.venice.hadoop.input.kafka.ttl;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * The policy controls the TTL behavior regarding how batch writes are treated.
+ * As of today (09/15/2022), only real-time write can be TTLed/supported.
+ */
+public enum TTLResolutionPolicy {
+  /**
+   * Reject batch writes as they don't contain RMD to perform TTL logic, so only real-time write. can be TTLed
+   */
+  REJECT_BATCH_WRITE(0),
```

Comment:
the name does not imply we are actually going to filter ttl based on rt write timestamp. May be `RT_WRITE_TIME` or something?

Reply:
Ack.